data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1010 @@ mission "Unfettered Jump Drive Trading"
+				goto end
+			label dangerous
+			`	"Worry not, matey. Respect only need be earned once. From me, at least. Do know there are other captains eager to challenge ye themselves, and most are stronger than me. Or, at least, they have more ships..."`
+				goto end
+			label weapons
+			`	"Aye, matey! The flashier yer weapons be when ye run a shot across their bow, the more likely these landlubbers be to strike colors and ditch their loot! No need to make these lily livered into shark food, just shiver their timbers a bit. Savvy? And not much else sets yer knees a knockin' quite like sending watching yer weapons go squiffy while yer energy levels drop straight ta Davy Jones' Locker! Always a good clap o' thunder for 'em!"`
+			label end
+			action
+				payment 250000
+			`	"Hopefully we see each other again, sometime." He and his crew seem quite busy with the repairs. "I transferred yer booty of <payment> to pay to repair any damage I may have done to yer ship. I be landing on the surface soon, to drown the sorrow of this humiliating defeat I have just suffered. I do not need nor want yer assistance in repairing, for that is a task best suited for the vanquished. I suppose ye may board me old ship to claim yer right o' salvage on me but ye wouldn't do that to a friend, would ye?"`
+			`	"If ye be feeling up to the challenge, there be other captains that would be willing to duel ye, if ye keep fighting fair, of course. None accept to fight more than a pair o' heavy men-o-war, some others, like me, still prefer fighting with a single vessel, when appropriate. Like me, they will wish to fight to test the strength of yer exotic technologies or strategies, savvy?"`
+			``
+			`	"Now land at once, we must officially conclude this duel, matey, else it will be worth no more than an empty chest!"`
+	to fail
+		has "unfettered: outcast"
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail If Killed"
+	on visit
+		dialog "You have not yet defeated the Unfettered Captain, and landing like this means you choose to delay the duel to another time."
+	on complete
+		conversation
+			`You land and the Captain joins you once he is done with repairs. He presents what looks like an aquarium as a reward for the duel.`
+			`	As you move closer to inspect it you are almost frightened to death by the huge scorpion looking creature rushing towards you.`
+			`	There is little time to think as the huge creature gets closer and closer!`
+			choice
+				`	(Run far away in case it escapes!)`
+					goto run
+				`	(Leap back to a respectable distance.)`
+				`	(Stay there.)`
+					goto stay
+				`	(Shoot it.)`
+					goto shoot
+				`	(Try to calm it.)`
+					goto calm
+			`	Surprisingly, no Unfettered look amused by the apparent cowardliness. "Fear is good, so long as it does not control you," one of them says.`
```

Comment:
And then you gain reputation with Si-Ra? Is there a missing goto here?

Reply:
```suggestion
			`	Surprisingly, no Unfettered look amused by the apparent cowardliness. "Fear is good, so long as it does not control you," one of them says.`
				goto reward
```